.github/workflows/test_service_static_checks.yml

```diff
@@ -1,27 +1,26 @@
 # See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
 
-name: 'Test Service: Code'
+name: 'Test Service: Static Checks'
```

Comment:
Actually nicer to have these separate indeed.

Maybe one day, we could set these to run "more often, quicker" and the others "only when PR is set to `ready-for-review` or something)...